torch/_logging/_internal.py

```diff
@@ -254,6 +257,18 @@ def set_logs(
             Whether to log communication operations and other debug info from pytorch distributed components.
             Default: ``logging.WARN``
 
+        dist_c10d (:class:`Optional[int]`):
+            Whether to log communication operations related debug info in pytorch distributed components.
```

Comment:
nit (before this PR): Should we stylize as "PyTorch Distributed"?

Reply:
Agree... Actually, PyTorch part is for sure, I am not sure if we should capitalize distributed. Or we just treat "PyTorch Distributed" as a term?

Reply:
Actually by checking PTD's code base, looks like we already using `PyTorch Distributed`, let's just keep this tradition. (Maybe change all PyTorch distributed to PyTorch Distributed as a follow-up)